Components/List/list.css

```diff
@@ -0,0 +1,58 @@
+.list__wrapper {
+  margin-right: 1rem;
+}
+
+.list-inline {
+  display: flex;
+  gap: 0.5rem;
```

Comment:
Use global variables for spacing for properties like margin, padding, and gap to maintain consistency across the website.